smart-contracts/genV2/run.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
```

Comment:
Mind either changing the file name to something more explicit and/or adding some larger comment block at the top explaining what this is for?